generators/ci-cd/templates/.gitlab-ci.yml.ejs

```diff
@@ -20,6 +20,18 @@
 image: jhipster/jhipster:v<%= jhipsterVersion %>
 <%_ } _%>
 
+<%_ if (prodDatabaseType === 'cassandra' || prodDatabaseType === 'redis' || prodDatabaseType === 'couchbase' || prodDatabaseType === 'neo4j') { _%>
```

Comment:
I think it should be `cacheProvider === 'redis'

Reply:
Yes, correct; thank you. Somehow I thought Redis was a DB type. :smile: